app/Console/Commands/GenerateAllianceHighscoreRanks.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OGame\Console\Commands;
+
+use Illuminate\Console\Command;
+use OGame\Enums\HighscoreTypeEnum;
+use OGame\Models\AllianceHighscore;
+
+class GenerateAllianceHighscoreRanks extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'ogamex:generate-alliance-highscore-ranks';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generates Alliance Highscore rank data';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): void
+    {
+        foreach (HighscoreTypeEnum::cases() as $type) {
+            $this->updateTypeRank($type);
+        }
+    }
+
+    private function updateTypeRank(HighscoreTypeEnum $type): void
```

Comment:
I would perhaps move this logic into the `GenerateHighscoreRanks` class, and have a new method called `updateAllianceRank` that way all the logic around highscore ranks are in one file then? But, up to you :)  

Thay way we only have one foreach of the enum cases, etc.